### ISSUE TITLE
WCAG

### ISSUE THREAD

**Issue by karolin-kivisikk** [TEDI-Design-System/design#184]
## Documentation (Zeroheight)
- [ ] **Accessibility page** is created - _includes instructions to meet accessibility requirements_

Komponendid kus WCAG jäi veel lahtiseks või ZH leht on loomata:
1. * https://github.com/TEDI-Design-System/design/issues/158 - Figmas jäi active state arutelu lahtiseks Futuse ideas lehel
2. * Eelmine mõjutab ka seda: https://github.com/TEDI-Design-System/design/issues/170
3. * https://github.com/TEDI-Design-System/design/issues/178
4. * https://github.com/TEDI-Design-System/design/issues/46
5. * https://tedi.zeroheight.com/styleguide/s/118912/p/649f97-progress-bar/b/15e86d
6. * https://tedi.zeroheight.com/styleguide/s/118912/p/27208b-timeline
7. * https://www.tedi.ee/1ee8444b7/p/125c5a-attachment 
8. * https://github.com/TEDI-Design-System/design/issues/147
9. * https://tedi.zeroheight.com/styleguide/s/118912/p/15bc4f-carousel/b/56169e
10. * https://www.figma.com/design/jWiRIXhHRxwVdMSimKX2FF/TEDI-READY-2.30.43?node-id=35122-129539&t=b1XRxrZpLBGs5Emf-4
11. Panen siia ühe juurde: https://github.com/TEDI-Design-System/design/issues/185

Anna Kärolinile märku kui need WCAG teemad tehtud sest ma uuendan ZH staatuste tabeli veerud ise ära.

**Comment by karolin-kivisikk** [comment]
Panen siia ühe juurde: https://github.com/TEDI-Design-System/design/issues/185

**Comment by karolin-kivisikk** [comment]
- Top header Figma - korras
- inverted infobutton Figma - korras
- accordion Figma - korras
- inverted ja brand closing button Figma - korras

**Comment by AngelicaSinisalu** [comment]
**NB! Task tehtud, aga kõik muudatused igaks juhuks publishimata ("hidden" olekus ZH's) + Atribuutide plokk puudu**
Inverted info button WCAG ja ZH ligipääsetavus üle vaadatud ja korras ✅
Inverted info button in sidenav WCAG ja ZH ligipääsetavus üle vaadatud ja korras ✅
Feedback form/Rating component WCAG ja ZH ligipääsetavus korras ✅
Search bar in header WCAG ja ZH ligipääsetavus üle vaadatud ja korras ✅
Progress bar WCAG ja ZH ligipääsetavus korras ✅
Timeline WCAG ja ZH ligipääsetavus korras ✅
Attachment WCAG ja ZH ligipääsetavus korras ✅
Carousel ZH ligipääsetavus korras ja WCAG üle kontrollitud ✅
Top header WCAG ja ZH ligipääsetavus korras ✅
+ LISAKS Accordion header WCAG üle kontrollitud + ZH ligipääsetavuse tabel juba olemas ✅

**Comment by karolin-kivisikk** [comment]
Irisele ülevaatamiseks:
* Rating: https://tedi.zeroheight.com/styleguide/s/118912/p/775eef-rating/b/80097d
* Progress bar: https://tedi.zeroheight.com/styleguide/s/118912/p/649f97-progress-bar/b/15e86d
* Timeline: https://tedi.zeroheight.com/styleguide/s/118912/p/27208b-timeline/b/657ca7
* Attachment: https://tedi.zeroheight.com/styleguide/s/118912/p/125c5a-attachment/b/50d9eb
* Carousel: https://tedi.zeroheight.com/styleguide/s/118912/p/15bc4f-carousel/b/276839
* Top header: https://tedi.zeroheight.com/styleguide/s/118912/p/002db0-top-header/b/39f2e5

**Comment by karolin-kivisikk** [comment]
@AngelicaSinisalu feedback: https://bitw3b.slack.com/archives/C063G1AQ50C/p1770386459576059?thread_ts=1769437996.617539&cid=C063G1AQ50C
